app/controllers/bookmarks_controller.rb

```diff
@@ -19,6 +19,7 @@ def create
     @bookmark = Bookmark.new(bookmark_params)
 
     if @bookmark.save
+      flash[:success] = "Bookmark successfully created!"
       redirect_to @bookmark
     else
```

Comment:
Yep, you've got to use `flash.now` when you use `render`
http://stackoverflow.com/questions/7133392/rails-flash-now-not-working